app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
Its recommended to use `find_by` instead of `where(...).first` for single record search. Read the note at the end of [this assignment](https://www.gotealeaf.com/lessons/a3a6fbb9/assignments/1566).